tests/issues/issue-1413.php

```diff
@@ -43538,8 +43538,8 @@
             'RenderingEngine_Version' => 'unknown',
             'RenderingEngine_Maker' => 'Apple Inc',
         ],
-        'lite' => true,
```

Comment:
This is the test that has that Sony A5000 device on Android 4.0 in it, by removing the "Android" platform, it doesn't get picked up by the Lite and Standard files any longer.